docs/sources/alerting/silences/create-silence.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Create a silence"
+description = "Add silence alert notification"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 450
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Create a silence
+
```

Comment:
No intro or before you begin?

Reply:
Not really - No useful information to add, that I can think of. Suggestions?